server/monitor.go

```diff
@@ -1128,15 +1128,9 @@ type monitorIPQueue struct {
 	InProgress int `json:"in_progress,omitempty"`
 }
 
-func (s *Server) HandleIPQueuesz(w http.ResponseWriter, r *http.Request) {
-	all, err := decodeBool(w, r, "all")
-	if err != nil {
-		return
-	}
-	qfilter := r.URL.Query().Get("queues")
-
+func (s *Server) Ipqueuesz(opts *IpqueueszOptions) map[string]monitorIPQueue {
```

Comment:
Better if this returned an exported type.

I added a set of response types here https://github.com/nats-io/nats-server/commit/8841131b99b3a8c004d9061370071026eae54c0f to help callers use these things, be good to have those for these also

Reply:
Done.